sdk/nodejs/__tests__/eventQueue.test.ts

```diff
@@ -1,19 +1,18 @@
-import { DVCPopulatedUser } from '../src/models/populatedUser'
+import { DVCPopulatedPBUser } from '../src/models/populatedPBUser'
 
-jest.mock('../src/request')
+jest.mock('../../js-cloud-server/src/request')
```

Comment:
this seems sus

Reply:
probably shouldn't be reaching into other packages to mock their internals

Reply:
how would I mock it otherwise?

Reply:
well it doesn't really make sense to move every request function to the other SDK, since half of them are only applicable to local bucketing. Things like fetching the config and stuff should stay in here